src/main/java/org/quartzpowered/network/pipeline/PacketHandler.java

```diff
@@ -41,9 +41,12 @@
 @Singleton
 @ChannelHandler.Sharable
 public class PacketHandler extends SimpleChannelInboundHandler<Packet> {
-    @Inject private Logger logger;
-    @Inject private SessionManager sessionManager;
-    @Inject private EventBus eventBus;
+    @Inject
+    private Logger logger;
+    @Inject
+    private SessionManager sessionManager;
+    @Inject
+    private EventBus eventBus;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.